.gitignore

```diff
@@ -10,3 +10,8 @@ docs/_build
 dist/
 local_examples/
 *.ipynb_checkpoints
+# files added after editable install (pip install -e . )
+pyroomacoustics.egg-info/
+pyroomacoustics/build_rir.c
+pyroomacoustics/build_rir.cpython-36m-x86_64-linux-gnu.so
```

Comment:
You will need to modify this for every version of python and architechture. Maybe just add *.so

Reply:
Good point!